### PR TITLE
C# allows multiple top level classes

### PR DESCRIPTION
**Pull request by neelkamath**
Here's a snippet you can quickly run: https://code.sololearn.com/cTzj2aWXBr3d/#

### CODE REVIEW

File: src/_guides/language/effective-dart/design.md [hunk 1]
```diff
@@ -532,7 +532,7 @@ doesn't know if any code outside of its view is using it.
 
 ### CONSIDER declaring multiple classes in the same library.
 
-Java and C# tie the organization of files to the organization of
+Java ties the organization of files to the organization of
```
Comment (kwalrath) [inline src/_guides/language/effective-dart/design.md]: Maybe we shouldn't tie this to Java so strongly (unless Java really is the only language that does this). Perhaps:

Some languages, such as Java, tie the organization...

**Comment by kwalrath** [comment]
@neelkamath thanks very much for the PR and code proof! @munificent do you have any feedback about the change?

File: src/_guides/language/effective-dart/design.md [hunk 2]
```diff
@@ -532,7 +532,7 @@ doesn't know if any code outside of its view is using it.
 
 ### CONSIDER declaring multiple classes in the same library.
 
-Java ties the organization of files to the organization of
+Some languages, such as Java, tie the organization of files to the organization of
```
Comment (kwalrath) [inline src/_guides/language/effective-dart/design.md]: 👍 Thanks!

**Comment by munificent** [comment]
Thanks!

**Pull request merged**